README.md

```diff
@@ -1,9 +1,14 @@
 # Link Extractor
 
-This repository is a [Docker](https://www.docker.com/) tutorial that illustrates some basics of Docker application containerization platform.
+Link Extractor app scrapes links from a given web page. This repository illistrates step by step approach to learn [Docker](https://www.docker.com/).
+
 It starts from running a very basic script and gradually leads to a multi-service container orchestration (AKA micro-service architecture).
-Follow instructions in the `README` files of various folders from `step0` to `step6` and explore corresponding code.
+
+## How to Use the repository
+1. [Play with Docker Classroom](https://training.play-with-docker.com/microservice-orchestration/) helps to start your docker journey without any additional overhead of configuring docker locally.
+2. Follow instructions in the `README` files of various folders from `step0` to `step6` and explore corresponding code.
```

Comment:
The two options are mutually exclusive, so I would start the second point a little differently.

```md
2. Alternatively, if running on a local Docker installation, follow instructions in the `README` files of various folders from `step0` to `step6` and explore corresponding code.
```